manifests/charts/base/templates/validatingwebhookconfiguration.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if or (eq .Values.revision "") (eq .Values.defaultRevision true) }}
```

Comment:
My problem is that 'base' is not supposed to be versioned. I don't think we're using revision in other places in base, and we should not touch base when installing a regular revision. Base is running with cluster-admin privs - rev install should only require istio-system permissions ( if we found a way to solve MutatingWebhook).

Reply:
We're going to have to pick somewhere that the abstraction leaks. ValidatingWebHook is in fact versioned--whenever the API objects change then it also has to change. The fact that we don't touch the VWH when installing a regular revision shouldn't be correct.

With this approach the only time the base _needs_ to be touched is on the first install. Afterwards you only need to touch base when the default revision changes, and that's logically part of base anyways.

Reply:
Good point - it means the cluster admin gets to decide what is the 'default' revision, and if we combine this with the revision shift label - will not touch it again.

However - if we do implement the proposal to use floating labels ( i.e. 'prod'/'stable' ) - do we still need this ? Can we just leave it as default, and use a service in istio-system to point to the correct revision ?

My concern was that 'base' is a singleton - you can't have base-prod and base-canary at the same time, not with help. 
It is probably ok if 'base' is pointing to a revision - but we shouldn't use the same naming, it will confuse users. For control plane, setting 'revision' will result in an isolated install of Istiod - all resources will have -revision suffix, etc. 

Can you remove Values.revision ( the confusing part - at least for me ) - and change 'defaultRevision' to a string, indicating the default revision ? I guess this will also allow us to move the MutatingWebhook for the default revision into base ?